.github/workflows/ci.yml

```
@@ -26,9 +26,10 @@ jobs:
         id: test-action
         uses: ./
         with:
-          mysql-host: ${{ secrets.MYSQL_HOST }}
-          mysql-user: ${{ secrets.MYSQL_USER }}
-          mysql-database: ${{ secrets.MYSQL_DATABASE }}
+          driver: 'mysql'
```

actionlint (1.7.4)

input "driver" is not defined in action "MySQL migration" defined at "./". available inputs are "mysql-database", "mysql-host", "mysql-user", "operation"

(action)